src/pip/_internal/utils/compat.py

```diff
@@ -47,7 +47,7 @@
 
 HAS_TLS = (ssl is not None) or IS_PYOPENSSL
 
-if sys.version_info >= (3, 4):
+if sys.version_info >= (3, 5):
```

Comment:
If we do not want to keep these as "the versions which introduced the feature", we should drop the minor number so it's clear this applies to all supported versions of Python 3.

Reply:
I think these changes should be done as part of a separate PR since they aren’t needed for the EOL, and that way it can be discussed separately.

Reply:
OK, I have marked the applicable items as resolved, we can address them separately.

Reply:
Oh, to clarify, I meant I agreed with you in that I don’t think they should be changed in the currently proposed way. But that if we do want to change them, it should be done separately. (So there is still something to be resolved for those comments until the changes are removed, IMO, unless I’m missing something.)